[gdcm] mac compilation error in medUtilities

On Jenkins mac, the compilation fails:

```
Undefined symbols for architecture x86_64:
  "gdcm::UIDGenerator::Generate()", referenced from:
      medUtilities::generateSeriesAndSOPInstanceId(medAbstractData*) in medUtilities.cpp.o
      medUtilities::generateUID() in medUtilities.cpp.o
      medUtilities::generateStudyIdAndInstanceUid(medAbstractData*) in medUtilities.cpp.o
ld: symbol(s) not found for architecture x86_64
```

--- a/src/layers/legacy/medUtilities/CMakeLists.txt
+++ b/src/layers/legacy/medUtilities/CMakeLists.txt
@@ -66,6 +66,7 @@ target_include_directories(${TARGET_NAME}
 
 target_link_libraries(${TARGET_NAME}
   ${QT_LIBRARIES}
+  ${ITK_LIBRARIES}
   medCore
   medCoreLegacy
   medVtkInria

